pkg/models/kr/keyrange.go

```diff
@@ -35,71 +40,189 @@ type KeyRange struct {
 //   - bool: True if kr is less than other, false otherwise.
 //
 // TODO : unit tests
-func CmpRangesLess(kr []byte, other []byte) bool {
-	if len(kr) == len(other) {
-		return string(kr) < string(other)
+func CmpRangesLessStringsDeprecated(bound string, key string) bool {
+	if len(bound) == len(key) {
+		return bound < key
 	}
 
-	return len(kr) < len(other)
+	return len(bound) < len(key)
 }
 
-// CmpRangesLessEqual compares two byte slices, kr and other, and returns true if kr is less than or equal to other.
-// The comparison is done by comparing the lengths of the slices first. If the lengths are equal, the function compares the byte values lexicographically.
-// Returns true if kr is less than or equal to other, false otherwise.
-//
-// Parameters:
-//   - kr: The first byte slice to compare.
-//   - other: The second byte slice to compare.
-//
-// Returns:
-//   - bool: True if kr is less than or equal to other, false otherwise.
-//
-// TODO : unit tests
```

Comment:
Move this docstring to respective method & update it instead of deleting

Reply:
done